pkg/backend/httpstate/client/api_endpoints.go

```diff
@@ -106,4 +106,7 @@ func init() {
 	addEndpoint("POST", "/api/stacks/{orgName}/{projectName}/{stackName}/{updateKind}/{updateID}/complete", "completeUpdate")
 	addEndpoint("POST", "/api/stacks/{orgName}/{projectName}/{stackName}/{updateKind}/{updateID}/events", "postEngineEvent")
 	addEndpoint("POST", "/api/stacks/{orgName}/{projectName}/{stackName}/{updateKind}/{updateID}/renew_lease", "renewLease")
+
+	// APIs for managing `PolicyPack`s.
+	addEndpoint("POST", "/api/orgs/{orgName}/policypacks", "publishPolicyPack")
```

Comment:
Are we missing the apply endpoint here?

Reply:
We're actually, technically, not. That is supposed to be a service-only endpoint, we only add it here in the commands because it is not exposed through the web UI. Once it is, we'll have to rip it out again.